src/sampling.jl

```diff
@@ -693,7 +739,13 @@ Noting `k=length(x)` and `n=length(a)`, this algorithm consumes ``O(k)`` random
 and has overall time complexity ``O(n k)``.
 """
 function naive_wsample_norep!(rng::AbstractRNG, a::AbstractArray,
-                              wv::AbstractWeights, x::AbstractArray)
+                              wv::AbstractWeights, x::AbstractArray)*
```

Comment:
```suggestion
                              wv::AbstractWeights, x::AbstractArray)
```

Reply:
Wow. Any idea why this doesn't trigger an error? :-/

Reply:
I think it is the same as if you have written:
```
function f()
    *
    some other code
end
```
And the operation just references the `*` function without calling it.